index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
title은 이렇게 수정해보면 어떨까요?

```suggestion
    <title>빨리 화면공유 안해?</title>
```

이러면 더 보기 좋을 것 같아요!